__tests__/app.test.js

```diff
@@ -89,3 +91,26 @@ describe("GET /api/articles", () => {
     return request(app).get("/api/nonexistent").expect(404);
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an object that contains all articles", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .then(({ body }) => {
+        body.comments.forEach((row) => {
+          expect(row).toMatchObject({
+            comment_id: expect.any(Number),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("404: endpoint not found", () => {
+    return request(app).get("/api/articles/700/comments").expect(404);
```

Comment:
This test is fine, but it should be in it's own describe block 👍 It's not specific to this endpoint, a user could type an incorrect endpoint at any time, so it just needs to be tested once in it's own describe